packages/application/src/status.ts

```diff
@@ -71,15 +72,28 @@ export class LabStatus implements ILabStatus {
    *
    * @returns A disposable used to clear the busy state for the caller.
    */
-  setBusy(): IDisposable {
+  setBusy(translator?: ITranslator): IDisposable {
+    translator = translator || nullTranslator;
+    const trans = translator.load('jupyterlab');
     const oldBusy = this.isBusy;
+    const ariaLiveRegion = document.getElementById('commands-aria-live');
+    let timeout: any;
```

Comment:
```suggestion
    let timeout: number;
```

Or `ReturnType<typeof setInterval>`